ideas/students-co-design-activity/index.html

```diff
@@ -0,0 +1,115 @@
+                   <button class="flc-prefsEditor-reset fl-prefsEditor-reset"><span class="fl-icon-undo"></span> Reset</button>
+               </span>
+           </div>
+       </div>
+       <!-- END markup for Preference Editor -->
+
+    <div id="body">
+      <div id="ideasHeader">
+
+            <div class="ideasHeaderContainer">
+                <div class="ideasHeaders"><h1><a href="/">Co-designing Inclusive Cities</a></h1>
+                <h2>Ideas in Progress</h2></div>
+                <div class="imageContainerIdeas"><img src="../../images/birds.png" class="img" alt="Birds" /></div>
+            </div>
+      </div>
+
+      <div id="ideasContent">
+
+          <div class="flc-toc-tocContainer toc"> </div>
+
+          <div class="ideasContentContainer">
+              <div class="ideasContentBlock">
+                  <span class="ideasDate">June 25, 2018</span>
+                  <h2>Students co-designing their dream neighbourhoods</h2>
+                  <p>The cities being built today are the cities our children will be living in as adults. However, children are seldom included in urban design and planning conversations and processes.</p>
+                  <p>As part of our efforts to include unheard voices in designing and planning of the Quayside project, we have approached primary and middle schools in Toronto to co-design the neighbourhood of their dreams. Students in grades 3, 5, 6, and 8 are going to join their teachers to work on a series of activities designed by the <a href="https://idrc.ocadu.ca/">IDRC (Inclusive Design Research Centre)</a> team, teachers, parents and inspired by the <a href="https://www.publicdeliberation.net/jpd/vol14/iss1/art8/">Me on the Map project</a>. These activities are accompanied by a <a href="https://docs.google.com/document/d/1FXPRpp6oafkjaGrcbp4RcVFf3mumyk7dQorMY9_zLhc/edit?usp=sharing">facilitation resource</a> that will guide teachers in performing the activities in the classroom on their own.</p>
+                  <p>During the activities, students are divided into small groups and given an activity board. The board includes a blank map of the neighbourhood and a list of 20 feature icons. Each group gets to discuss the features and vote for the top ten to include in their neighbourhood. Once the features are selected, groups can exchange their activity boards to consider pros and cons of the features selected by their peers. Once done, groups can move back to their original activity board, discuss the pros and cons and decide whether they want to change any of their features based on their peers’ feedback. As a final activity, students are given a development proposal that encourages them to think more critically about other important urban issues, such as the ethics of gathering, owning and using digital data.</p>
+
+                  <div id="Image">
+
+                      <div class="ImageContainer">
```

Comment:
I think we don't have a class `"ImageContainer"` (uppercase I) only lowercase "i" imageContainer. So I think we can remove the `<div class="ImageContainer">` markup.